src/lib/helpers.ts

```diff
@@ -394,3 +394,19 @@ export function replaceString(str: string, stringA: string, stringB: string) {
   const regExp = new RegExp(escapedStringA, "g");
   return str.replace(regExp, stringB);
 }
+
+// determineRbacMode determines the RBAC mode to use based on the cli and the module's config
+export function determineRbacMode(opts: { rbacMode?: string }, cfg: { pepr: { rbacMode?: string } }): string {
```

Comment:
This doesn't seem like a function that's generalized enough to need to live in a `helpers` collection.  It also only appears to be used in _one_ place (the `build.ts` file).

Would there be anything stopping you from moving this helper back into `build.ts` and just using it directly?  Much more cohesive over there & would help prevent the `helpers.ts` file from becoming a junk drawer for single-use funcs.

Reply:
It first lived there but I realized that we cannot unit test it there. Would the unit test still live in helpers?

Reply:
After ruminating on this issue, considering many alternatives I came to the conclusion to create a folder `cli-helpers/build.ts`.

Adding a coverageIgnorePath in the jest config had adverse affects. Babel warnings occurred in the console and the coverage dropped by ~15%, more strictly defined paths would have been necessary as well as an adjustment of other configs to suppress the babel warning or increase a threshold.

```
[BABEL] Note: The code generator has deoptimised the styling of /Users/cmwylie19/not-pepr/pepr/node_modules/@kubernetes/client-node/dist/gen/types/ObjectParamAPI.js as it exceeds the max of 500KB.
[BABEL] Note: The code generator has deoptimised the styling of /Users/cmwylie19/not-pepr/pepr/node_modules/@kubernetes/client-node/dist/gen/types/ObservableAPI.js as it exceeds the max of 500KB.
[BABEL] Note: The code generator has deoptimised the styling of /Users/cmwylie19/not-pepr/pepr/node_modules/@kubernetes/client-node/dist/gen/apis/AppsV1Api.js as it exceeds the max of 500KB.
```

Separate jest configs caused validation warnings

```
● Validation Warning:

  Unknown option "collectCoverage" with value true was found.
  This is probably a typing mistake. Fixing it will remove this message.

  Configuration Documentation:
  https://jestjs.io/docs/configuration
 ```
 
 We would have needed to run jest from node through node_modules and add an `--experimental-vm-modules` from our test.
 
 This approach seems to satisfy the requirement and and provide us room to grow in the future.